templates/prescription/general_fragment.html

```diff
@@ -1,24 +1,29 @@
 {**
- * Prescription General Fragment
- *
- * @package   OpenEMR
- * @link      http://www.open-emr.org
- * @author    Brady Miller <brady.g.miller@gmail.com>
- * @copyright Copyright (c) 2017 Brady Miller <brady.g.miller@gmail.com>
- * @license   https://github.com/openemr/openemr/blob/master/LICENSE GNU General Public License 3
- *}
+* Prescription General Fragment
+*
+* @package   OpenEMR
+* @link      http://www.open-emr.org
+* @author    Brady Miller <brady.g.miller@gmail.com>
+* @author    Sherwin Gaddis <sherwingaddis@gmail.com>
+* @copyright Copyright (c) 2017 Brady Miller <brady.g.miller@gmail.com>
+* @copyright Copyright (c) 2019 Sherwin Gaddis <sherwingaddis@gmail.com>
+* @license   https://github.com/openemr/openemr/blob/master/LICENSE GNU General Public License 3
+* added dose and units to display.
+*}
 <table>
-  {if empty($prescriptions)}
-        <tr class='text'>
-                <td>&nbsp;&nbsp;{xlt t='None'}</td>
-        </tr>
-  {/if}
-	{foreach from=$prescriptions item=prescription}
-  {if $prescription->get_active() > 0}
-	<tr class='text'>
-		<td>{$prescription->drug|text}</td>
-		<td>{$prescription->get_dosage_display()|text}</td>
-	</tr>
-  {/if}
-	{/foreach}
+    {if empty($prescriptions)}
+    <tr class='text'>
+        <td>&nbsp;&nbsp;{xl t='None'|escape:'html'}</td>
```

Comment:
revert above line (also note below where you've changed prior code that should also be reverted)